app/emoji.go

```diff
@@ -5,6 +5,9 @@ package app
 
 import (
 	"bytes"
+	"errors"
+	"fmt"
+	"github.com/mattermost/mattermost-server/v5/store"
```

Comment:
This should be in the other import group.

And while here, please also make "image/color/palette" part of this group.